vendor/bat-native-ledger/src/bat/ledger/internal/wallet/create.cc

```diff
@@ -299,7 +299,7 @@ void Create::RegisterPersonaCallback(
   ledger_->SetAutoContributeEnabled(true);
   braveledger_state::SetPaymentId(ledger_, payment_id);
   braveledger_state::SetAnonymousCardId(ledger_, card_id);
-  braveledger_state::SetFetchOldBalanceEnabled(ledger_, false);
+  //braveledger_state::SetFetchOldBalanceEnabled(ledger_, false);
```

Comment:
@NejcZdovc I wasn't quite sure how to deal with this one. It seems that we reset this flag to false after registering the persona, which interferes with these browser tests.

Reply:
what we can do is wrap this line in `!ledger::is_testing`

Reply:
Perfect, will do.

Reply:
Fixed.